RoomControlService.cs

```diff
@@ -45,7 +45,8 @@ public static List<roomControl> UpdateColor()
                     }
                     isDeleteRead.Add(new roomControl() { RoomNo = roomNo });
                 }
-                
+                readerQuery.Close();
+                connectionIsDelete.Close();
                 return isDeleteRead;
```

Comment:
bunun öncesinde eğer fasle yani dolu olan bir oda yok ise veritabanında hiç değer atamadan buradan çıkmayı sağlayacak bir yöntem(mesela sorgu) uygulamayı planlıyorum. aksi takdirde  asılsız bir hata vermiş olacak gibi geliyor. 
sizce nasıl olur böyle bir yöntem eklemek ? @zencifil

Reply:
oncelikle bir sorum var: UpdateColor neden silinmemis odalarin listesini donduruyor?

Reply:
Bunu soyle degistireyim; UpdateColor neden color update etmiyor da liste donuyor?

Reply:
buradaki isimlendirmem hatalı olmuş. yaptığım işlem doğrultusunda değil de  kullanılacağı yer için bir isimlendirme yapmışım. 
burada QueryİsDelete gibi bir isimlendime yapmam gerekirdi.